ext/active_span_source.py

```diff
@@ -49,3 +53,21 @@ def active_span(self, loop=None):
         # get the current running Task for this loop and return the ActiveSpan
         task = asyncio.Task.current_task(loop=loop)
         return getattr(task, '__active_span', None)
+
+    def deactivate(self, span, loop=None):
```

Comment:
I definitely like having the deactivation here - for the Java API it had to remain, because of the `Continuation` part, in the `ActiveSpan` part - but having it all in the `ActiveSpanSource` feels better.